experiments/chaos/cspc_pool_failure/pool_expansion_inprogress/README.md

```diff
@@ -0,0 +1,30 @@
+## Experiment Metadata
+
+| Type       | Description                                           | Storage      | Applications | K8s Platform |
+| ---------- | ----------------------------------------------------- | ------------ | ------------ | ------------ |
+| CHAOS      | Restart the pool pod while CSPC pool expansion is in progress | OpenBS cStor | Any          | Any          |
+
+## Entry-Criteria
+
+- K8s nodes should be ready.
+- cStor CSPC should be created.
+- Application should be deployed using with volume provisioned through CSI provisioner.
+- Uncliamed BlockDevices should be available to expand the pool.
+
+## Exit-Criteria
+
+- Pool expansion should be expanded successfully and the application is still up and running.
```

Comment:
Pool should be expanded successfully and the application should be still up and running without any disruption.